### PR TITLE
TNO-1861 Add frontpages session

### PR DESCRIPTION
**Pull request by fbarreta**
Just added the menu option to load the frontpages by it's own.

Some changes to be aware off:

- Filter option in "_libs/net/dal/Services/ContentService.cs_" was change to 10 (Front page images) the previous was 11 (Corporate Calendar). It's it safe to keep it hardcoded and it will reflect the same in production ?
- Later work will be captured in an upcoming ticket since @bbjornholt have the idea of having this session as a photo gallery.

### CODE REVIEW

File: libs/net/dal/Services/ContentService.cs [hunk 1]
```diff
@@ -179,7 +179,7 @@ public async Task<IPaged<API.Areas.Services.Models.Content.ContentModel>> FindFr
     {
         var productQueries = new List<Func<QueryContainerDescriptor<API.Areas.Services.Models.Content.ContentModel>, QueryContainer>>();
         var today = DateTime.Today.ToUniversalTime();
-        productQueries.Add(q => q.Raw(@"{""match"": {""productId"": 11}}"));
+        productQueries.Add(q => q.Raw(@"{""match"": {""productId"": 10}}"));
```
Comment (Fosol) [inline libs/net/dal/Services/ContentService.cs:182]: We can't hardcode this.  It shouldn't have been hardcoded in the first place.
Additionally, we don't want searches to use NEST going forward.
We have a way to configure setting values that are essentially constants for each environment.
Talk to @jdtoombs about using an Elasticsearch query from the web app, and how to use Settings.

**Review by Fosol — CHANGES_REQUESTED** [review]

File: libs/net/dal/Services/ContentService.cs (hunk 1 above)
Comment (fbarreta) [reply libs/net/dal/Services/ContentService.cs:182]: That's nice. I'll work on this.

File: libs/net/dal/Migrations/1.8.80/Up/PostUp/00-Settings.sql
Comment (Fosol) [inline libs/net/dal/Migrations/1.8.80/Up/PostUp/00-Settings.sql:1]: This isn't an ideal solution due to the configurable nature of filters and their names.  Remove the DB migration.  Add a toast error to the page to inform the user that a setting needs to be configured.  We have other examples of this.

File: app/subscriber/src/features/todays-front-pages/TodaysFrontPages.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      try {
+        const res: unknown = await findContentWithElasticsearch(filter, false);
+        setResults(res);
+      } catch {}
+    },
+    [findContentWithElasticsearch],
+  );
+
+  React.useEffect(() => {
+    const id = settings.find((s) => s.name === 'FrontpageFilter')?.value;
```
Comment (Fosol) [inline app/subscriber/src/features/todays-front-pages/TodaysFrontPages.tsx:36]: Add a `toast.error("FrontpageFilter setting needs to be configured.")`

**Review by Fosol — CHANGES_REQUESTED** [review]
It's looking very close.  Great work.  Just a couple minor changes.
- Remove the DB migration.
- Add a toast to inform user that we need to configure the setting.

File: libs/net/dal/Migrations/TNOContextModelSnapshot.cs
Comment (Fosol) [inline libs/net/dal/Migrations/TNOContextModelSnapshot.cs:1]: Remove this file

**Review by Fosol — CHANGES_REQUESTED** [review]

File: libs/net/dal/Migrations/TNOContextModelSnapshot.cs
Comment (Fosol) [inline libs/net/dal/Migrations/TNOContextModelSnapshot.cs:1]: Your branch history is off.  This file should not be part of your PR.

**Review by Fosol — CHANGES_REQUESTED** [review]

**Review by Fosol — APPROVED** [review]

**Review by Fosol — CHANGES_REQUESTED** [review]
The build shows some linting issues.

**Review by Fosol — APPROVED** [review]

**Pull request merged**